getting-started/git-boot-camp.rst

```diff
@@ -669,3 +669,65 @@ Examples of useful commands:
 * Set the browser::
 
       $ gh config set browser <browser-path>
+
+
+Git worktree
+------------
+
+When working on several version branches at once, experienced Git users can
+take advantage of Git's worktree to check out several branches at once.
+
+Setting up Git worktree
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With an existing CPython clone (see :ref:`clone-your-fork`), rename the
+``cpython`` directory to ``main`` and move it into a new ``cpython``
+directory, so we have a structure like:
+
+.. Generated with: tree -L 1 -d cpython
+
+.. code-block:: text
+
+   cpython
+   └── main (.git is here)
+
+Next, create worktrees for the other branches::
+
+   $ cd cpython/main
+   $ git worktree add -b 3.11 ../3.11 upstream/3.11
+   $ git worktree add -b 3.12 ../3.12 upstream/3.12
+
+This gives a structure like this, with the code for each branch checked out in
+its own directory:
+
+.. code-block:: text
+
+   cpython
+   ├── 3.11
+   ├── 3.12
+   └── main
+
+Using Git worktree
+^^^^^^^^^^^^^^^^^^
+
+List your worktrees, for example::
+
+   $ git worktree list
+   /Users/my-name/cpython/main  b3d24c40df [main]
+   /Users/my-name/cpython/3.11  da1736b06a [3.11]
+   /Users/my-name/cpython/3.12  cf29a2f25e [3.12]
+
+Change into a directory to work from that branch. For example::
+
+   $ cd ../3.12
+   $ git checkout my-3.12-bugfix-branch
+   ...
+   $ cd ../main
+   $ git checkout my-feature-branch
+   ...
+
```

Comment:
Are more steps needed to configure pulls from upstream and pushed to origin (personal fork)?

Reply:
I've committed Ezio's suggestion that shows a more concrete example of pushing and switching back, how does [`3b26368` (#1255)](https://github.com/python/devguide/pull/1255/commits/3b2636881b40ac42246a70fee975a6957b216a85) look?